### PR TITLE
feat(trg-7.06): add about page

### PR DESCRIPTION
**Pull request by fabiodmota**
## Description
This pull request will add the about page to our dashboard it also the dynamic application of the vars on the license guide .

https://github.com/eclipse-tractusx/vas-country-risk/issues/64

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-    "@mui/material": "5.10.7",
+    "@mui/x-data-grid": "5.17.26",
+    "@mui/types": "7.2.5",
     "@testing-library/jest-dom": "^6.1.4",
     "@testing-library/react": "14.1.0",
     "@testing-library/user-event": "^14.5.1",
     "axios": "1.6.1",
-    "cx-portal-shared-components": "^0.4.5",
+    "buffer": "^6.0.3",
+    "cx-portal-shared-components": "0.4.5",
```
Comment (evegufy) [inline package.json:18]: why do you need the deprecated lib as well?
@catena-x/portal-shared-components should be the leading one.

File: public/index.html [hunk 2]
```diff
@@ -7,7 +7,7 @@
       window.ENV = {
         REACT_APP_COUNTRY_RISK_API: "http://localhost:8080",
         REACT_APP_AUTH_URL: "http://localhost:8080/auth",
-        REACT_APP_PORTAL_FRONTEND: "http://localhost:8080",
+        REACT_APP_PORTAL_FRONTEND: "http://localhost:3000",
```
Comment (evegufy) [inline public/index.html:10]: not sure what this change has to do with the about page, but if you want to do it, I suggest to replace the anchor [here](https://github.com/eclipse-tractusx/vas-country-risk/blob/f90173fbb9b611b632342e531d06114709e2bc1c/scripts/inject-dynamic-env.sh#L29) as well, other wise the dynamic env injection won't work for this value.

File: src/components/dashboard/Footer/orange-background-head.svg
Comment (evegufy) [inline src/components/dashboard/Footer/orange-background-head.svg:1]: I suggest to add a license file, essentially you can copy the one [here](https://github.com/eclipse-tractusx/portal-assets/blob/main/public/assets/images/frame/orange-background-head.svg.license)

File: src/components/dashboard/Footer/orange-background-full.svg
Comment (evegufy) [inline src/components/dashboard/Footer/orange-background-full.svg:1]: I suggest to add a license file, essentially you can copy the one [here](https://github.com/eclipse-tractusx/portal-assets/blob/main/public/assets/images/frame/orange-background-full.svg.license)

**Review by evegufy — CHANGES_REQUESTED** [review]

**Comment by evegufy** [comment]
@fabiodmota due to the updated [package.json](https://github.com/eclipse-tractusx/vas-country-risk/pull/65/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) can you confirm that you checked Dependencies and all licensed are vetted by the Dash Tool?

**Comment by fabiodmota** [comment]
> @fabiodmota due to the updated [package.json](https://github.com/eclipse-tractusx/vas-country-risk/pull/65/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) can you confirm that you checked Dependencies and all licensed are vetted by the Dash Tool?

yes i can confirm 
image

**Review by evegufy — APPROVED** [review]

**Pull request merged**